operator/pkg/operator.go

```diff
@@ -150,9 +150,20 @@ func (o *Operator) ProcessNewTaskCreatedLog(newTaskCreatedLog *servicemanager.Co
 	switch provingSystemId {
 	case uint16(common.GnarkPlonkBls12_381):
 		verificationKey := newTaskCreatedLog.Task.VerificationKey
-		VerificationResult := o.VerifyPlonkProof(proof, pubInput, verificationKey)
+		VerificationResult := o.VerifyPlonkProofBLS12_381(proof, pubInput, verificationKey)
```

Comment:
We could change it to:
`verificationResult := o.verifyPlonkProof(proofBytes, pubInputBytes, verificationKeyBytes, ecc.BLS12_381)`
See comment below.